plugins/login-plugin/src/Components/AssetFlow/index.js

```diff
@@ -27,6 +31,24 @@ import {
 import MESSAGES from "./Config";
 
 const AssetFlow = (props) => {
+  const memoizedOnChange = useCallback(tvosRemoteHandler, []);
```

Comment:
You need to pass all your external variables to the watcher, otherwise, all the values are locked to the initial state. 

In this case, looks like you need to pass `screen` variable to the watcher.